examples/usb-serial/src/main.rs

```diff
@@ -0,0 +1,75 @@
+#![no_std]
+#![no_main]
+#![feature(type_alias_impl_trait)]
+#![feature(used_with_arg)]
+
+use riot_rs::{
+    debug::log::info,
+    embassy::{
+        embassy_usb, make_static,
+        usb::{UsbBuilderHook, UsbDriver},
+    },
+};
+
+use embassy_usb::{
+    class::cdc_acm::{CdcAcmClass, State},
+    driver::EndpointError,
+};
+
+#[riot_rs::config(usb)]
+fn usb_config() -> embassy_usb::Config<'static> {
+    let mut config = embassy_usb::Config::new(0xc0de, 0xcafe);
```

Comment:
There are codes for use with test devices, and while I don't know off my head where to look these up, but they're probably made up.

Good candidates are [1209](https://pid.codes/1209/) / 7D01 (RIOT Test PID) or 1298 / 000[0-f] (generic test PIDs)

Reply:
"code cafe" is certainly made up. :) This is copied from upstream. I'll open an issue for default USB ids.

Reply:
https://github.com/future-proof-iot/RIOT-rs/issues/339